traffic_ops/app/lib/API/Cachegroup.pm

```diff
@@ -164,6 +166,7 @@ sub update {
 		latitude                       => $params->{latitude},
 		longitude                      => $params->{longitude},
 		parent_cachegroup_id           => $params->{parentCachegroupId},
+		fallback_to_closest            => $params->{fallbackToClosest},
```

Comment:
fallbackToClosest needs to be added to the Traffic Portal

form.cacheGroup.tpl.html